src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/clusterinteraction/ClusterInteractionFragment.kt

```diff
@@ -71,6 +73,7 @@ class ClusterInteractionFragment : Fragment() {
 
   companion object {
     private const val TAG = "ClusterInteractionFragment"
+    var devicePtr = 0L
```

Comment:
Like in ClusterDetailFragment, we should pass this to other fragments instead of making it a static field.